index.js

```diff
@@ -293,33 +288,39 @@ client.on(Events.InteractionCreate, async (interaction) => {
                 const messageTracker = await generalChat.messages.fetch(MESSAGE_TRACKER_ID);
 
                 // Grab current message tracker reviews 
-                const currentMessageTrackerContent = messageTracker.content;
+                const messageTrackerContent = messageTracker.content;
```

Comment:
Hmm now that the variable's been renamed, is it still clearer by making it into a separate variable?